File: CryBot.Core/Exchange/Models/Candle.cs
using Bittrex.Net.Objects;

using CryBot.Core.Strategies;

using System;

namespace CryBot.Core.Exchange.Models
{
    public class Candle
    {
        public string Currency { get; set; }

        public DateTime Timestamp { get; set; }

        public decimal High { get; set; }

        public decimal Low { get; set; }

        public decimal Open { get; set; }

        public decimal Close { get; set; }

        public decimal Volume { get; set; }
        
        public TickInterval Interval { get; set; }
        
        public TradeAdvice EmaAdvice { get; set; }
    }
}

File: CryBot.Web/ClientApp/app/models/Candle.ts
import { TradeAdvice } from "./TradeAdvice";

export class Candle {
    market: string = "";
    open: number = 0;
    close: number = 0;
    high: number = 0;
    low: number = 0;
    timestamp: Date;
    emaAdvice: TradeAdvice;
}